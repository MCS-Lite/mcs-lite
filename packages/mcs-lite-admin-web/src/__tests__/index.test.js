import * as Constants from '../index';

jest.mock('react-dom');

it('should return Constants correctly', () => {
  expect(Constants).toMatchSnapshot();
});
