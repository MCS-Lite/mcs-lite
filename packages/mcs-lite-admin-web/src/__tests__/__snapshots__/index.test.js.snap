// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return Constants correctly 1`] = `
Object {
  "BASENAME": "/admin",
  "DEFAULT_LOCALE": "zh-TW",
  "ROOT_ID": "root",
}
`;
